src/insert_pad.cpp

```diff
@@ -91,8 +91,12 @@ static void update_pooling(const instruction_ref& input, const instruction_ref&
     std::copy(pads_l.begin(), pads_l.end(), padding.begin() + 2);
     std::copy(pads_r.begin(), pads_r.end(), padding.begin() + kdims + 2 + 2);
 
-    // maxpool uses lowest value for padding
-    float pad_val = std::numeric_limits<float>::lowest();
```

Comment:
Wasn't this also a problem for average pooling with count_include_pad?

Reply:
not sure about it